bevy_lint/docs/how-to/release.md

```diff
@@ -6,10 +6,12 @@
 2. Replace `[Unreleased]` heading with the version with the format `[vX.Y.Z] - YYYY-MM-DD`.
 3. Update the `**All Changes**` link to compare from `main` to the new tag `lint-vX.Y.Z`. (E.g. `lint-v0.1.0...main` to `lint-v0.1.0...lint-v0.2.0`.)
 4. Review the [migration guide](../../MIGRATION.md) and ensure all breaking / significant changes from the previous version are documented.
-4. Remove the `-dev` suffix from the version in [`Cargo.toml`](../../Cargo.toml) and the compatibility table in [`README.md`](../../README.md).
+5. Remove the `-dev` suffix from the version in [`Cargo.toml`](../../Cargo.toml) and the compatibility table in [`README.md`](../../README.md).
     - Please ensure that [`Cargo.lock`](../../../Cargo.lock) also updates!
-6. Commit all of these changes and open a pull request.
-7. Merge the PR once a core Bevy maintainer approves it with no outstanding issues from other contributors.
+6. Replace `--branch main` in [`action.yml`](../../action.yml) with `--tag lint-vX.Y.Z`.
+    - The [`linter-action.yml`](../../../.github/workflows/linter-action.yml) workflow may fail as the tag does not exist yet. This is fine!
```

Comment:
I'm not happy with this, but I'm not really sure how to get around it. Should we create the tag early and update it for the release? Or should we just let the workflow fail?

Reply:
Let's try the simplest workflow first and then refine it as we go